man/data.table-options.Rd

```diff
@@ -0,0 +1,120 @@
+\name{data.table-options}
+\alias{data.table-options}
+\alias{data.table.options}
+\alias{datatable.options}
+\alias{datatable-options}
+
+\title{Global Options for the data.table Package}
+
+\description{
+  The data.table package uses a number of global options to control its
+  behavior. These are regular R options that can be set with options()
+  and retrieved with getOption(). For example:
+  \preformatted{
+    # Get the current value of an option
+    getOption("datatable.print.topn")
+
+    # Set a new value for an option
+    options(datatable.print.topn = 10)
+  }
+  This page provides a comprehensive, up-to-date list of all user-configurable
+  options. NB: If you're reading this on the web, make sure the version numbers match with what you have installed.
+}
+
+\section{Printing Options}{
+  See \code{\link{print.data.table}} for a full description of printing data.tables.
+  \describe{
+   \item{\code{datatable.print.topn}}{An integer, default \code{5L}. When a data.table is printed,
+     only the first topn and last topn rows are displayed.}
+   \item{\code{datatable.print.nrows}}{An integer, default \code{100L}. The total number of rows
+     to print before the topn logic is triggered.}
+   \item{\code{datatable.print.class}}{A logical, default \code{FALSE}. If \code{TRUE}, the class of
+     each column is printed below its name.}
+   \item{\code{datatable.print.keys}}{A logical, default \code{FALSE}. If \code{TRUE}, the table's
+     keys are printed above the data.}
+   \item{\code{datatable.show.indices}}{A logical, default \code{TRUE}. A synonym for `datatable.print.keys` for historical reasons.}
```

Comment:
Something that's tough -- remembering that `` ` `` is not supported like markdown in .Rd files. In fact, that should be a second Rd linter, but let's mark that for follow-up. It's not trivial since we have to exclude example code (and other common sources of false positives).